app/models/dfe_sign_in/user.rb

```diff
@@ -0,0 +1,9 @@
+class DfeSignIn::User < ApplicationRecord
```

Comment:
Should we be validating the presence of things like name, email, organisation, etc?